_sass/frogtek-theme.scss

```diff
@@ -231,7 +232,28 @@ footer {
 
 .empowering {
   font-family: 'ShadowsIntoLight';
-  font-size: 3em;
+  font-size: 3.5vw;
+}
+
+@media screen and (min-width: 1200px) {
```

Comment:
Same here: `@media screen and (min-width: 1140px) {`

Reply:
Ready @durey